dev.py

```diff
@@ -727,9 +727,10 @@ def run_lcov(dirs):
         print("Capturing lcov info...")
         LCOV_OUTPUT_FILE = os.path.join(dirs.build, "lcov.info")
         LCOV_OUTPUT_DIR = os.path.join(dirs.build, "lcov")
+        BUILD_DIR = str(dirs.build)
         lcov_cmd = [
             "lcov", "--capture",
-            "--directory", dirs.build,
+            "--directory", BUILD_DIR,
```

Comment:
I think the error you faced was because of this change missing. I fixed it on my Linux cloud but pushed it from my macOS (and this change wasn't there). Apologies for the troubles.

Reply:
That works for me now, thanks.